test-utils/store-validator-bin/Cargo.toml

```diff
@@ -1,22 +0,0 @@
-[package]
-name = "store-validator-bin"
```

Comment:
Why did you choose to merge store-validator-bin into store-validator lib crate? This brings a few dependencies into the dependant crates. I prefer to have many crates instead of fat crates introducing unreasonable dependencies.

Reply:
Discussed offline. I renamed store-validator-bin into store-validator, and previous store-validator is a part of chain now.